config/config.wikimedia.yaml

```diff
@@ -13,7 +13,7 @@ worker_heap_limit_mb: 250
 
 # Logger info
 logging:
-  level: info
+  level: debug
```

Comment:
Why?

Reply:
This is only used locally in development, and I needed it to see what was happening with loading schemas.  I used to have this at debug for development, but sometime some patch switched it to info.  It doesn't matter.